csharp-advanced-topics/PointersInC#/PointersInC#/Method/Methods.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace PointersInCSharp.Method;
+
+public static class Methods
+{
+    public static unsafe void SwapChars(char* p1, char* p2) => (*p2, *p1) = (*p1, *p2);
+    public static unsafe void DoubleOddValues(int[] numbers)
+    {
+        fixed (int* arrayPtr = numbers)
+        {
+            for (int i = 0; i < numbers.Length; i++)
```

Comment:
Can you use `var` here instead?
```suggestion
            for (var i = 0; i < numbers.Length; i++)
```

Reply:
Changed to var.